__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require('supertest');
+const app = require('../app.js');
+const db = require('../db/connections.js');
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/test-data/index.js');
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() =>{
+    return db.end();
+})
+
+describe('3: /api/categories', () => {
+    test('GET - 200: responds with array of category objects, each with slug and description properties', () => {
+        return request(app)
+            .get('/api/categories')
+            .expect(200)
+            .then(({body}) => {
+                body.categories.forEach((category) => {
+                    expect(category).toMatchObject({
+                        slug: expect.any(String),
+                        description: expect.any(String),
+                    })
+                })
+            });
+    });
+});
+
+describe('4: /api/reviews', () => {
+    test('GET: 200 - responds with reviews array of review objects, with correct properties', () => {
+        return request(app)
+            .get('/api/reviews')
+            .expect(200)
+            .then(({body}) => {
+                body.reviews.forEach((review) => {
```

Comment:
Will still need to assert the length of the array here - otherwise the forEach will not run